.gitignore

```diff
@@ -1,3 +1,3 @@
-!.gitignore
+#.gitignore
```

Comment:
- [ ] [Optional] This Pull Request should have a descriptive title telling you what changes are inside, you can change it so it reflects the content of the PR